crates/mdman/Cargo.toml

```diff
@@ -16,7 +16,7 @@ serde_json.workspace = true
 url.workspace = true
 
 [dev-dependencies]
-pretty_assertions.workspace = true
+snapbox.workspace = true
```

Comment:
Nobody uses `pretty_assertions` so it can also be removed from `workspace.dependencies` I believe.

Reply:
:)  Sometimes its easier to delete a line than to hand write the parameters (since I don't have completions configured)